dotCMS/src/main/java/com/dotmarketing/portlets/workflows/ajax/WfActionAjax.java

```diff
@@ -1,6 +1,7 @@
 package com.dotmarketing.portlets.workflows.ajax;
 
 import com.dotcms.workflow.form.WorkflowActionForm;
+import com.dotcms.workflow.form.WorkflowReorderActionStepForm;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.workflow.form.WorkflowReorderActionStepForm'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)